setup.py

```diff
@@ -30,25 +31,28 @@ def local_open(fname):
 for directory in os.walk('example/templates'):
     example_dirs.append(directory[0][8:] + '/*.*')
 
+with local_open(os.path.join('pages', '__init__.py')) as f:
+    exec(f.read(), {}, pages)
+
 url_schema = 'http://pypi.python.org/packages/source/d/%s/%s-%s.tar.gz'
-download_url = url_schema % (package_name, package_name, pages.__version__)
+download_url = url_schema % (package_name, package_name, pages['__version__'])
 
 install_requires = [
-    'django-mptt>=0.8.3,<0.9.0',
-    'six>=1.10.0,<1.11.0',
-    'Pillow>=3.2.0,<3.3.0',
-    'tqdm>=4.4.0,<4.5.0',
-    'django-taggit>=0.18.1,<0.19.0',
-    'requests>=2.9.0,<3.0.0',
+    'django-mptt>=0.8.3',
+    'six>=1.10.0',
+    'Pillow>=3.2.0',
+    'tqdm>=4.4.0',
+    'django-taggit>=0.18.1',
+    'requests>=2.9.0',
```

Comment:
Unless you know that a version breaks, don't prevent the users of `django-page-cms` from using the version(s) of their choice. E.g., I would really love to use `Pillow==3.3.1` but because of `django-page-cms`, I'm stuck with `^3.2.0`.

As a library editor, you shouldn't define the concrete dependencies other will use.

> You want a specific set of dependencies that you want to fetch from a specific location and you know that you should have abstract dependencies in your `setup.py` and concrete dependencies in your `requirements.txt` but you don’t want to need to maintain two separate lists which will inevitably go out of sync.
> 
> https://caremad.io/2013/07/setup-vs-requirement/

Relax ;-)